lib/tls.js

```diff
@@ -796,13 +797,15 @@ function onnewsession(key, session) {
  */
 
 function SecurePair(credentials, isServer, requestCert, rejectUnauthorized,
-                    options) {
+                    options,
+                    pskCallback) {
```

Comment:
It would probably be best to keep the signature the same as Node.js uses so perhaps instead of adding an optional pskCallback argument we can just include psdkCallback as a value in options?